code/modules/reqs/supplypacks.dm

```diff
@@ -796,6 +796,11 @@ WEAPONS
 	contains = list(/obj/item/ammo_magazine/pistol/standard_pistol/smart_pistol)
 	cost = 10
 
+/datum/supply_packs/weapons/pepperball_purger_ammo
```

Comment:
Why did you give it import gun level ammo pricing? The 20 you get roundstart should realistically be enough for the whole round, but this just makes it dirt cheap to restock on top